integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -184,6 +189,161 @@ case class AlterTableCompaction(alterTableModel: AlterTableModel) extends Runnab
   }
 }
 
+/**
+ * Command for Alter Table Add & Split partition
+ * Add is a special case of Splitting the default partition (part0)
+ * @param alterTableSplitPartitionModel
+ */
+case class AlterTableSplitPartition(alterTableSplitPartitionModel: AlterTableSplitPartitionModel)
+  extends RunnableCommand {
```

Comment:
with `SchemaProcessCommand` and `DataProcessCommand`
split `run` method into `processSchema` and `processData`